license.txt

```diff
@@ -0,0 +1,12 @@
+
+
+<!--
+Copyright (c) 2018 by freeCodeCamp (https://codepen.io/freeCodeCamp/pen/MJjpwO)
+
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+-->
```

Comment:
Required changes:
- [ ] I think your review would have been more complete you added an explanation of why the changes were necessary. The reviews, in my opinion, are supposed to be more conversation-friendly and less commanding, don't you agrees? 😄 
- [ ] I think it would have been great if you use did use the `GitHub inline comments feature` at least once to leave review comments. For this project kindly make sure to leave the comment under a line or block of code that fits the context the comment is about. This not only makes the comment complete but understandable as well. I will try to invite you to check this [lesson on how to add inline comments](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request).
- [ ] Be sure to also leave at least two comments. That way the review will hold more water and be more deliberate. I will also urge you to try and use a couple of emojis and/or gifs in the review. I think they always make the code review more human-friendly :alien::robot: 
- [ ] Kindly carry out these changes in your review. When done, please submit another code review request. We have here a link to [the part of the lesson about inline comments and what have you](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) that you can use.